analyzer/codechecker_analyzer/makefile.py

```diff
@@ -138,8 +138,12 @@ def __get_ctu_pre_analysis_cmds(self, action):
         cmds.append(' '.join(cmd))
 
         # Get command to create CTU index file.
+        # FIXME Even if clang-extdef-mapping supports pch files as parameters,
+        # feed the source files rather. This way, the below 'sed' command is
+        # still meaningful.
```

Comment:
You lost me here a little. What is the context?

Reply:
> The FIXME states the current hack, or what the fix should be? I'm not in the clear. 
`makefile.py` had been architectured with a technical debt.

> Shouldn't the FIXME state that we should use the ast_dump_path method? 
Ok, I am updating the comment with that.

> What is preventing us from doing it? Why does this divergence exist?
That would require further changes in `makefile.py` which is out of the scope of this patch, in my opinion. Addressing the mentioned technical debt should be done in a follow up patch.

> Also, what you wrote here could be in the code as well, right?
Could be, yes.